### PR TITLE
Add both handlers for queries from new Payment API

### PR DESCRIPTION
**Pull request by jeffffc**
Added PreCheckoutQueryHandler and ShippingQueryHandler

### CODE REVIEW

File: telegram/precheckoutquery.py [hunk 1]
```diff
@@ -88,3 +88,7 @@ def to_dict(self):
         data['from'] = data.pop('from_user', None)
 
         return data
+
+    def answer(self, *args, **kwargs):
+        """Shortcut for ``bot.answerPreCheckoutQuery(update.callback_query.id, *args, **kwargs)``"""
```
Comment (jh0ker) [inline telegram/precheckoutquery.py]: The docstring has `callback_query` instead of `pre_checkout_query`

File: telegram/shippingquery.py [hunk 2]
```diff
@@ -73,3 +73,7 @@ def to_dict(self):
         data['from'] = data.pop('from_user', None)
 
         return data
+
+    def answer(self, *args, **kwargs):
+        """Shortcut for ``bot.answerShippingQuery(update.callback_query.id, *args, **kwargs)``"""
```
Comment (jh0ker) [inline telegram/shippingquery.py]: same as above

File: telegram/update.py [hunk 3]
```diff
@@ -132,6 +143,12 @@ def effective_user(self):
         elif self.callback_query:
             user = self.callback_query.from_user
 
+        elif self.shipping_query:
+            user = self.shipping_query.from_user
+
+        elif self.pre_checkout_query:
+            user = self.pre_checkout_query
```
Comment (bomjacob) [inline telegram/update.py]: Missing a `.user` here?

Comment (jh0ker) [inline telegram/update.py]: `.from_user`

**Comment by jh0ker** [comment]
Thanks :) Can you open the PR again, to the master branch?

**Pull request merged**